src/main/java/com/epam/lowcost/controller/UserController.java

```diff
@@ -0,0 +1,66 @@
+package com.epam.lowcost.controller;
+
+import com.epam.lowcost.model.User;
+import com.epam.lowcost.service.UserServiceImpl;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+import java.time.LocalDateTime;
+import java.util.Map;
+
+@Controller()
+@RequestMapping(value = "/user")
+
+public class UserController {
+
+    @Autowired
+    UserServiceImpl userService;
+
+    @GetMapping(value = "/all")
+    public String listAllUsers(Model model)
```

Comment:
I see that you have more than one code style, it's not very cool)) Please use ctrl+L and choose common plugin for you team (for example IDEA's code style)

Reply:
Adjusted style.